resources/js/app.js

```diff
@@ -45,6 +75,45 @@ function updateComments(){
 
 }
 
+function deleteComments(){
+
+    let commentDeleteForms = document.querySelectorAll(".comment-delete-form");
+
+    commentDeleteForms.forEach(form =>{
+
+        form.addEventListener("submit", async (event) => {
+            event.preventDefault();
+
+            let commentId = `${form.querySelector('#commentId').value}`;
+            let urlStringDelete = `/comments/${commentId}`;
+            let commentInstanceId = `comment-${commentId}`;
+            let commentInstance = document.getElementById(commentInstanceId);
+            var csrf = document.querySelector('meta[name="csrf-token"]').content;
+
+            try {
+
+                const response = await fetch(urlStringDelete, {
+                    method: "DELETE",
+                    headers: {
+                        'X-CSRF-Token': csrf,
+                    }
+                });
+
+                if (response.status === 200) {
+                    commentInstance.remove();
+                }
+
+            } catch (error) {
+                console.log(error)
+            }
+
+        });
+
+    });
+
+}
+
 document.addEventListener("DOMContentLoaded", (event) => {
     updateComments();
```

Comment:
change these method signatures to a more appropriate name, like initializeCommentForms and initializeDeleteCommentButtons